Dockerfile

```diff
@@ -1,23 +1,67 @@
-# AUTHOR: Roberto Vitillo
-# DESCRIPTION: Mozilla's Airflow container
-# BUILD: docker build --rm -t mozdata/telemetry-airflow
-# SOURCE: https://github.com/mozilla/telemetry-airflow
-
-FROM puckel/docker-airflow:1.7.1.3
-MAINTAINER vitillo
-
-USER root
-RUN apt-get update -yqq && \
-    apt-get install -yqq python-pip python-mysqldb && \
-    pip install boto3 && \
-    pip install airflow[async] && \
-    pip install airflow[password] && \
-    pip install retrying
-
-ADD ansible/files/airflow/airflow.cfg ${AIRFLOW_HOME}/airflow.cfg
-ADD ansible/files/airflow/entrypoint.sh ${AIRFLOW_HOME}/entrypoint.sh
-ADD ansible/files/airflow/replace_env.py ${AIRFLOW_HOME}/replace_env.py
-RUN chown airflow:airflow ${AIRFLOW_HOME}/airflow.cfg
-
-USER airflow
-ADD dags/ /usr/local/airflow/dags/
+FROM python:2-slim
+MAINTAINER Mozilla Telemetry
+
+ENV PYTHONUNBUFFERED 1
+
+ENV AWS_REGION us-west-2
+# ENV AWS_ACCESS_KEY_ID
+# ENV AWS_SECRET_ACCESS_KEY
+
+ENV SPARK_BUCKET telemetry-spark-emr-2
+ENV AIRFLOW_BUCKET telemetry-test-bucket
+ENV PRIVATE_OUTPUT_BUCKET telemetry-test-bucket
+ENV PUBLIC_OUTPUT_BUCKET telemetry-test-bucket
+
+ENV EMR_KEY_NAME mozilla_vitillo
+ENV EMR_FLOW_ROLE telemetry-spark-cloudformation-TelemetrySparkInstanceProfile-1SATUBVEXG7E3
+ENV EMR_SERVICE_ROLE EMR_DefaultRole
+ENV EMR_INSTANCE_TYPE c3.4xlarge
+
+ENV AIRFLOW_HOME /app
+ENV AIRFLOW_AUTHENTICATE False
+ENV AIRFLOW_BROKER_URL redis://redis:6379/0
```

Comment:
Ah, simple, Redis is operationally a lot less complex locally and both AWS (ElastiCache) and Heroku offer service for it for production.

Celery supports Redis as a 1st level broker and Airflow doesn't use any of its more complex features that would require a more advanced broker such as RabbitMQ. In fact, I believe the "CeleryExecutor" is pretty limited in how it can be configured, so it's running almost with the default Celery configuration.

In short: for simplicity's sake